Bottom: c7ec313ec30e76edebd064b143821e612ad7ec56
Top:    17c2139b555fe3be6156ee28a3caa2bdf9eb9913
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-01-04 20:07:16 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
